docs/topics/development/search.rst

```diff
@@ -4,41 +4,36 @@
 How does search on AMO work?
 ============================
 
-.. note::
-
-  This is documenting our current state of how search is implemented in addons-server.
-  We will be using this to plan future improvements so please note that we are
-  aware that the process written below is not perfect and has bugs here and there.
-
-  Please see https://github.com/orgs/mozilla/projects/17#card-10287357 for more planning.
-
```

Comment:
Comparing this doc with https://github.com/mozilla/addons-server/blob/master/src/olympia/search/filters.py reveals no major changes since this doc was written.

I've linked the old GitHub project in https://mozilla-hub.atlassian.net/browse/AMO-245 since it contains a few issues in backlog that might still be relevant today.